webvtt_test.go

```diff
@@ -121,6 +121,25 @@ func TestWebVTTWithTimestampMap(t *testing.T) {
 	err = s.WriteToWebVTT(b)
 	assert.NoError(t, err)
 	assert.Equal(t, `WEBVTT
+X-TIMESTAMP-MAP=MPEGTS:180000,LOCAL:00:00:00.000
```

Comment:
I don't think we need to test the `.Add()` method here 🤔 

I would only assert the first item's start at on the first webvtt read body, and I would also assert the timestamp map header is written properly.

Reply:
This was a pre-existing test that originally validated that the timestamp map's offset was properly applied if the parsed WebVTT contained a map. When my changes broke the test I figured I'd just replicate the behavior that was already there with the new code. I can remove this test if you'd like.

Reply:
Adjusted, thanks 👍